Remove passport card option

hursey013 opened the pull request:
Fixes #818

This removes the `Passport Card` option from the `Foreign associations > U.S. passport information` UI, also updates the validation and i18n files to reflect the changes. The database field for Passport Card still remains and will be documented in a [separate issue](https://github.com/18F/e-QIP-prototype/issues/840) should it need to be reenabled in the future.

**Before:**
<img width="697" alt="screen shot 2018-09-20 at 2 28 15 pm" src="https://user-images.githubusercontent.com/1178494/45839065-45d43a00-bce1-11e8-83ed-46703a9cce57.png">

**After:**
<img width="691" alt="screen shot 2018-09-20 at 2 24 38 pm" src="https://user-images.githubusercontent.com/1178494/45839012-20473080-bce1-11e8-9dd8-3927b8e26b09.png">

apburnes commented on src/components/Section/Foreign/Passport/Passport.jsx:
```diff
@@ -311,7 +283,6 @@ Passport.defaultProps = {
   HasPassports: {},
   suggestedNames: [],
   reBook: '^[a-zA-Z]{1}[0-9]{6,9}$',
```
I happened to have my passport near by and it has a type letter and then a 9 digit passport number.  When I entered that schema into the form, I cannot input all 9 digits into the form. Looks like the regex only allows 9 places including the letter. Should this be updated to `7-10` to allow the initial letter space?

apburnes replied:
Looks like the regex is fine. Just need to make the max length of `10`

hursey013 replied:
The help text is also a little misleading as well, the passport number in my book doesn't technically begin with a letter... I guess that is referring to the `Type` letter?

apburnes replied:
Should we turn it into two inputs? One is a letter type and the other is a 6-9 digit number?

hursey013 replied:
This seems inaccurate as well `Passports will start with the letter A.` https://github.com/18F/e-QIP-prototype/pull/839/files#diff-f3e54c23a039ce2474f26a85429c3bc8R79, @ryanhofdotgov do you know the source of that help text? The paper SF-86 makes no mention of needing a letter at all.

apburnes commented on src/components/Section/Foreign/Passport/Passport.jsx:
```diff
@@ -215,38 +209,16 @@ export default class Passport extends SubsectionElement {
                   )}
-                  pattern={re}
+                  pattern={this.props.reBook}
                   maxlength="9"
```
Increase to `10`

ryanhofdotgov commented:
@apburnes @hursey013  Yeah, I don't know where that help text came from. We should remove it.

Any time we are changing the fields, we should look at the validation matrix to double-check. On page 6 it says:
* If issue date >= 1/1/1990, verify it is a exactly nine characters, alphanumeric
* if less than 1/1/1990, any ASCII string

Here is another .gov reference: 
https://www.e-verify.gov/about-e-verify/e-verify-data/e-verify-enhancements/june-2011

It seems to contradict the validation matrix. Let me do some tests against e-QIP before we call this "done".

ryanhofdotgov commented:
I've checked e-QIP, it does what the validation matrix says. Let's implement that. To be clear, we are not soliciting the passport type code, even though that might be implied from the previous help text.

Also, on second read, that e-verify.gov document doesn't really contradict what we see in the validation matrix, just isn't as precise as to the condition a less than nine character passport number might come into play.

hursey013 commented:
Alright, updated with more accurate help text and adjusted the regex to accommodate alphanumeric strings between 6 and 9 characters, loosening it a bit to account for 9 digit number only strings (like mine).

hursey013 commented:
Ready for another look - takes passport issue date into consideration in choosing the proper number validation (before 1990, alphanumeric string, no length limit. after 1990, alphanumeric string, 6-9 chars).
